Zotero/Scenes/Detail/ItemDetail/Views/ItemDetailCollectionViewHandler.swift

```diff
@@ -628,19 +633,16 @@ final class ItemDetailCollectionViewHandler: NSObject {
                 let isProcessing = state.backgroundProcessedItems.contains(attachment.key)
                 return attachmentRow(for: attachment, isProcessing: isProcessing)
             })
-
-            if !viewModel.state.data.isAttachment {
+            if !viewModel.state.data.isAttachment && state.library.metadataEditable && state.library.filesEditable {
```

Comment:
Yes, I noticed that you can have metadata not editable and files editable. But if you want to create an attachment for example, you need both editable (because an item - metadata is created first and only then you can upload an actual file), so I added both checks there. Added convenience var in e0da345e10df783df4e986e560e3636dbd84c942